### DISCUSSION TITLE
[FEATURE REQUEST] Git blame on "Files changed"

### DISCUSSION THREAD

**Discussion by oliveryasuna** [community/community#122362] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be very useful if, when reviewing a PR on the "Files changed" tab, hovering a line would show the Git blame. That way, for PRs will multiple authors (e.g., commit co-authors or multiple commits from different authors), reviewers can know exactly who to ask about specific changes.

**Comment by hvsw** [comment]
Would also be useful to see the commit message, not only the author, on a change. There are some other requests about it.

https://github.com/orgs/community/discussions/12968
https://github.com/orgs/community/discussions/31585
https://github.com/orgs/community/discussions/7465

**Comment by pccayaan** [comment]
It's quite unbelievable we still don't have this feature live on GH yet. It's 2026 already!
